### DISCUSSION TITLE
LMSTUDIO API Key (OpenAI compat)

### DISCUSSION THREAD

**Discussion by StackShard** [logancyang/obsidian-copilot#565] [Q&A]
I’m running LMStudio on a beefy machine and set an API key. Where do I specify the API key when choosing LMStudio?

**Comment by logancyang** [comment]
Then you should use OpenAI proxy base url instead of `LMSTUDIO (LOCAL)`, set your API key in OpenAI API key, and pick any OpenAI model in the dropdown as a placeholder.
